### ISSUE TITLE
Jump doesn't occur when a pop-up shows up during its jump operation

### ISSUE THREAD

**Issue by manaswis** [ProjectSidewalk/SidewalkWebpage#702]
If you reach the end of route (i.e. "end of route" message shows up) and you completed the distance where the 'contributing a lot' message is triggered, the user is not jumped to the new location when the user clicks the jump message, and hence, pano doesn't change. But internally the new task is assigned, so the following "Uh-oh.." message shows up when user moves.

![image](https://user-images.githubusercontent.com/2873216/27313799-e33b22e8-553d-11e7-92cf-3866c7a9cca6.png)

My hunch: It has to do with disabling interactions for pop-ups. Would have to look into if appropriate lock was obtained before jumping so that simultaneous operations such as the pop-up showing up (which disables interactions) doesn't interfere with the previous task.

FIX: The pop-up should be shown after the jump operation is completed.

**Comment by jonfroehlich** [comment]
Nice catch @manaswis. Should we also tag these things with a priority level or is anything tagged Relaunch Testing instantly high priority. :)

**Comment by manaswis** [comment]
Yep, high priority. :)

Unless there is a bug that we don't want to solve for Relaunch, we can mark them as priority low.

**Comment by jonfroehlich** [comment]
> Unless there is a bug that we don't want to solve for Relaunch, we can mark them as priority low.

Agree.

**Comment by misaugstad** [comment]
Agreed with what @manaswis said. It seems like it has to do with clicking the button while we are in the middle of taking a step (where all interactions are disabled).

**Comment by jonfroehlich** [comment]
I reached the end of a route and clicked on the bar in bottom right hand corner to 'jump to a new location' but I didn't actually jump. You can tell I didn't jump because the screen says I've been "moved to a new location" but you can see the labels that I had just added at that intersection.

![image](https://user-images.githubusercontent.com/1621749/54033890-6185ac80-416a-11e9-89a1-48fcff1f9289.png)

@misaugstad, any thoughts?

**Comment by misaugstad** [comment]
Mapillary/infra3D integration bought about a rework of the Explore page routing code, so I expect that this issue should be resolved.
